nautobot_ui/src/components/core/Plugins.jsx

```diff
@@ -23,16 +23,20 @@ function get_components() {
     for (const [plugin_name, import_promise] of Object.entries(NautobotPlugins)) {
         import_promise.then((value) => {
             if (value?.default?.view_overrides) {
+                // eslint-disable-next-line
```

Comment:
It wants return values from nested functions. The correct fix is what I did below in `nautobot_ui/src/views/generic/ObjectRetrieve.jsx`. Instead of updating a list as you iterate, returning a value from the `map` calls.

```
src/components/core/Plugins.jsx
  Line 26:83:  Array.prototype.map() expects a return value from arrow function  array-callback-return
  Line 27:74:  Array.prototype.map() expects a return value from arrow function  array-callback-return
  Line 34:95:  Array.prototype.map() expects a return value from arrow function  array-callback-return
  Line 36:48:  Array.prototype.map() expects a return value from arrow function  array-callback-return
```